src/main/java/com/epam/helpers/ErrorMessagesProvider.java

```diff
@@ -45,4 +45,8 @@ public static String getLessThan30DaysErrMessage() {
     public static String getExistedSubjectNameErrMessage() {
         return PROPERTIES_READER.getProperty("existedSubjectName");
     }
+
+    public static String getSelectionError() {
```

Comment:
Please change the method name to "getSelectionErrMessage", because all method in this class are written as I say

Reply:
done